tests_with_odoo/recipe.cfg

```diff
@@ -4,6 +4,7 @@ parts = odoo
 [odoo]
 addons = git https://github.com/oca/web parts/web ${odoo:release} depth=1
 apply-requirements-file = True
+write-requirements-file = requirements.txt
```

Comment:
I would very much like to avoid `requirements.txt` terminology here to avoid any implication to the users of oca.recipe.odoo that this file will at some point actually be used to pin versions of installed packages. I'd be much happier here with something like `installed_versions.txt`, or as a compromise, `installed_requirements.txt`.

Reply:
Actually I'm fine with the name of the parameter because it indicates the format of the file.

Reply:
I just realize that the automatism why we do this in the first place only works when the file is called `requirements.txt` - what do you think about the [comment I added](https://github.com/OCA/oca.recipe.odoo/pull/7/commits/cc836ad69c14976ea5d32101026af01051252a20)?

Reply:
Perfect!